conn/raft_server.go

```diff
@@ -120,20 +121,22 @@ func (p *proposals) Done(key string, err error) {
 	pd.Ch <- err
 }
 
-func (w *RaftServer) GetNode() *Node {
+func (w *RaftServer) getNode() *Node {
 	w.nodeLock.RLock()
 	defer w.nodeLock.RUnlock()
 	return w.Node
 }
 
+// RaftServer stores a RAFT node along with a lock.
 type RaftServer struct {
 	nodeLock sync.RWMutex // protects Node.
```

Comment:
rename `nodeLock` to `lock` or simply `m`

Reply:
Done.